blocks/DropArea/addDropzone.js

```diff
@@ -113,6 +124,12 @@ export function addDropzone(desc) {
     desc.element.removeEventListener('drop', onDrop);
     FINAL_EVENTS.forEach((eventName) => {
       window.removeEventListener(eventName, onFinalEvent, false);
+      dragImageSrc = null;
     });
   };
 }
+
+window.addEventListener('dragstart', (e) => {
```

Comment:
I think it would be better to define this handler inside `addDropzone`

Reply:
Yes sure.